test/goldens/end2end/angular_components-0.10.0.json_report.md

```diff
@@ -39,230 +39,82 @@ No example found.
-[`analyzer`]: https://pub.dev/packages/analyzer
-[`angular_ast`]: https://pub.dev/packages/angular_ast
-[`angular_compiler`]: https://pub.dev/packages/angular_compiler
-[`args`]: https://pub.dev/packages/args
-[`built_value`]: https://pub.dev/packages/built_value
-[`charcode`]: https://pub.dev/packages/charcode
-[`checked_yaml`]: https://pub.dev/packages/checked_yaml
-[`cli_repl`]: https://pub.dev/packages/cli_repl
-[`code_builder`]: https://pub.dev/packages/code_builder
-[`convert`]: https://pub.dev/packages/convert
-[`crypto`]: https://pub.dev/packages/crypto
-[`csslib`]: https://pub.dev/packages/csslib
-[`dart2_constant`]: https://pub.dev/packages/dart2_constant
-[`dart_internal`]: https://pub.dev/packages/dart_internal
-[`dart_style`]: https://pub.dev/packages/dart_style
-[`file`]: https://pub.dev/packages/file
-[`front_end`]: https://pub.dev/packages/front_end
-[`glob`]: https://pub.dev/packages/glob
-[`http`]: https://pub.dev/packages/http
-[`http_parser`]: https://pub.dev/packages/http_parser
-[`json_annotation`]: https://pub.dev/packages/json_annotation
-[`kernel`]: https://pub.dev/packages/kernel
-[`matcher`]: https://pub.dev/packages/matcher
-[`node_interop`]: https://pub.dev/packages/node_interop
-[`node_io`]: https://pub.dev/packages/node_io
-[`package_config`]: https://pub.dev/packages/package_config
-[`package_resolver`]: https://pub.dev/packages/package_resolver
-[`pedantic`]: https://pub.dev/packages/pedantic
-[`pub_semver`]: https://pub.dev/packages/pub_semver
-[`pubspec_parse`]: https://pub.dev/packages/pubspec_parse
-[`sass`]: https://pub.dev/packages/sass
-[`source_gen`]: https://pub.dev/packages/source_gen
-[`source_maps`]: https://pub.dev/packages/source_maps
-[`source_span`]: https://pub.dev/packages/source_span
-[`stack_trace`]: https://pub.dev/packages/stack_trace
-[`stream_transform`]: https://pub.dev/packages/stream_transform
-[`string_scanner`]: https://pub.dev/packages/string_scanner
-[`term_glyph`]: https://pub.dev/packages/term_glyph
-[`tuple`]: https://pub.dev/packages/tuple
-[`typed_data`]: https://pub.dev/packages/typed_data
-[`watcher`]: https://pub.dev/packages/watcher
-[`yaml`]: https://pub.dev/packages/yaml
-
-Found 9 issues. Showing the first 2:
-
-<details>
-<summary>
-The constraint `^5.1.0` on angular does not support the stable version `6.0.0`.
-</summary>
+* Could not run pub outdated: `pub get` failed: 
 
-Try running `dart pub upgrade --major-versions angular` to update the constraint.
-</details>
-<details>
-<summary>
-The constraint `^2.1.0` on angular_forms does not support the stable version `3.0.0`.
-</summary>
+ ```
+The current Dart SDK version is {{sdk-version}}.
 
-Try running `dart pub upgrade --major-versions angular_forms` to update the constraint.
-</details>
+Because observable >=0.1.1 <=0.22.1+3 requires SDK version <2.0.0 or >=2.0.0-dev.23.0 <2.0.0 and observable >=0.22.1+4 depends on dart_internal ^0.1.1, observable >=0.1.1 requires dart_internal ^0.1.1.
```

Comment:
Seems no version of angular_components resolve with 2.12.0.
I wonder if we could find another package to use that has similar characteristics.